pkg/components/validate_headers.go

```diff
@@ -75,10 +83,19 @@ func (*ValidateHeaderConfigComponent) Settings() *ValidateHeaderConfig {
 
 // New generates the middleware.
 func (*ValidateHeaderConfigComponent) New(_ context.Context, conf *ValidateHeaderConfig) (func(tripper http.RoundTripper) http.RoundTripper, error) {
+	if strings.TrimSpace(conf.Split) == "" { // revert to default
+		conf.Split = ","
```

Comment:
Question: Do we want the default to still split? Shouldn't the default not split? This one is debatable...

Reply:
Default to no split will make this a breaking change.

Reply:
@ensary @iamnotbrandonchan This being a breaking change could be fine if we create a new version, I think defaulting to split could lead to some scenarios where it is undesirable

Reply:
👍 , especially if we will update the existing deployment in a timely fashion.

Reply:
it's possible we could do the breaking change in a separate PR though